core/src/main/java/org/mule/runtime/core/privileged/routing/RoutingResult.java

```diff
@@ -42,13 +46,17 @@ public RoutingResult(Map<String, Message> successfulRoutesResultMap, Map<String,
   }
 
   public static RoutingResult routingResultWithException(Map<String, Message> successfulRoutesResultMap,
-                                                         Map<String, Pair<Error, EventProcessingException>> failedRoutesErrorWithExceptionMap) {
```

Comment:
preserve the naming (failedRoutesErrorWithExceptionMap)

Reply:
done